README.md

```diff
@@ -32,12 +32,13 @@ The build process requires the following packages:
 - build-essential
 - python3
 - binutils-mips-linux-gnu (libultra* only)
+- wget
 
-Under Debian / Ubunutu you can install them with the following commands:
+Under Debian / Ubuntu you can install them with the following commands:
 
 ```bash
 sudo apt update
-sudo apt install build-essential python3
+sudo apt install build-essential python3 wget
```

Comment:
We don't use wget do we? i thought that was just to download the versions of gcc for matching